### ISSUE TITLE
Change setup to pyproject

### ISSUE THREAD

**Issue by nyggus** [nyggus/makepackage#17]
Pyuthon is moving from setup.py files to pyproject.toml files. `makepackage` should too.

**Comment by Pato546** [comment]
I think this is a good direction for makepackage, will try and work on that.

**Comment by nyggus** [comment]
I' am happy to see that. I lack time to contribute to the package the way it deserves.

**Comment by Pato546** [comment]
I have created a pull request which includes a pyproject.toml file for packaging.

**Comment by nyggus** [comment]
Thanks. Please gimme some time to review this - and I will try to make this time reasonable. Please let me know in what environments (OSs) you checked this.

**Comment by Pato546** [comment]
On windows

**Comment by nyggus** [comment]
And which Python version? We need to check it in all main Python 3.6 and higher versions. I will check also on Linux.

I added a comment to the merge request. We need a pyproject.toml file not only in the package itself, but also in the packages it creates.

**Comment by Pato546** [comment]
yeah yeah, I will add that, but I want to know if nothing is breaking as of now?

**Comment by Pato546** [comment]
I have added the pyproject.toml file when `makepackage` is invoked. Let me know of any feedback. I am using Python 3.9
